src/main/web_src/src/pages/gruppeOversikt/FinnPerson.tsx

```diff
@@ -0,0 +1,135 @@
+import React, { useState } from 'react'
+//@ts-ignore
+import AsyncSelect from 'react-select/async'
+import { useAsyncFn } from 'react-use'
+import { Redirect } from 'react-router-dom'
+import useBoolean from '~/utils/hooks/useBoolean'
+import { TpsfApi } from '~/service/Api'
+import NavButton from '~/components/ui/button/NavButton/NavButton'
+import DollyModal from '~/components/ui/modal/DollyModal'
+import { Label } from '~/components/ui/form/inputs/label/Label'
+import { Søkeknapp } from 'nav-frontend-ikonknapper'
+import { AsyncFn } from 'react-use/lib/useAsync'
+
+type FinnPerson = {
+	naviger: Function
+}
+
+type Option = {
+	value: string
+	label: string
+}
+
+type Person = {
+	ident: string
+	fornavn: string
+	mellomnavn?: string
+	etternavn: string
+}
+
+type Respons = {
+	value: {
+		data: {
+			gruppe?: {
+				id: number
+			}
+			error?: string
+			message?: string
+		}
+	}
+}
+
+export default function FinnPerson({ naviger }: FinnPerson) {
+	const [isFinnModalOpen, openFinnModal, closeFinnModal] = useBoolean(false)
+	const [redirectToGruppe, setRedirect] = useBoolean()
+
+	const [ident, setIdent] = useState(null)
+	const [gruppe, setGruppe] = useState(null)
+	const [feilmelding, setFeilmelding] = useState(null)
+
+	const [options, fetchOptions]: AsyncFn<any> = useAsyncFn(async tekst => {
+		const { data }: any = await TpsfApi.soekPersoner(tekst)
+		const options: Array<Option> = []
+		data.map((person: Person) => {
+			const navn = person.mellomnavn
+				? `${person.fornavn} ${person.mellomnavn} ${person.etternavn}`
+				: `${person.fornavn} ${person.etternavn}`
+			options.push({
+				value: person.ident,
+				label: `${person.ident} - ${navn.toUpperCase()}`
+			})
+		})
+		return options
+	}, [])
+
+	const handleChange = (tekst: string) => {
+		fetchOptions(tekst)
+		setFeilmelding(null)
+	}
+
+	const handleClose = () => {
+		setIdent(null)
+		setGruppe(null)
+		setFeilmelding(null)
+		closeFinnModal()
+	}
+
+	const navigerTilPerson = () => {
+		naviger(ident).then((response: Respons) => {
+			if (response.value.data.error) {
+				setFeilmelding(response.value.data.message)
+			} else {
+				setGruppe(response.value.data.gruppe.id)
+				setRedirect()
+			}
+		})
+	}
+
+	if (redirectToGruppe) return <Redirect to={`/gruppe/${gruppe}`} />
+
+	return (
+		<>
+			{/* 
```

Comment:
Dobbelt opp med kommentartegn? `{/* // */}` . Ser det samme på ts-ignore lenger ned

Reply:
Dette var en liten hack for å unngå en bug hvor //@ts-ignore inne i komponenter ble rendra som tekst. Men nå ser det jammen meg ut til at de har fiksa det, så da slipper vi det stygge oppsettet 😅